LCM/scripts/calcPythonPath.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+if [ "$#" -ne 0 ]
+then
+	# Checking which python version is available 
+	# python3
+	if [[ -z $(python2 --version 2>&1 | grep 'not found') ]]
```

Comment:
isnt there a better way to check this ?>

Reply:
Based on my understanding, this is the cleanest way to test the python environment.